### DISCUSSION TITLE
NFT流动性的想法

### DISCUSSION THREAD

**Discussion by BradMoonUESTC** [rebase-network/hello-world#49] [Ideas]
NFT流动性设计及思考
1、当前现状
截止2021年9月，nft碎片化分为两部分，即以碎片化方式来区分，分为集合法与指数化：
* 集合法-创建接受不同NFT的基金，然后将该基金代币化，用户可以购买基金代币来接触原始NFT集合
* 指数化-通过创建具有相似价格的 NFT基金来创建一个价格底线

2、当前处理方式的问题
这两种处理方式都是将NFT看作一个整体进行碎片化，然而这产生了问题，即碎片化后的NFT非同质性依旧很高，其原因在于碎片化后的NFT依然保留各自的属性和特殊的数据，这样就导致了无法像ERC20一样做到真正的流动性

3、一种新型的NFT碎片化设计

3.1 在当前的NFT市场中，大部分NFT的创作方式是一个个trait的组合，一个项目的nft中，trait可能达到了几十个甚至上百个，而发售时是将trait组合起来进行发售

3.2 trait拆分的想法，我们可以将一个未上市的nft以trait拆分的形式上市，或是将现有的nft进行trait拆分，以下为例：
【假设有trait: A,B,C,D,E作为一个nft的身体；12345作为NFT的配饰】
则根据排列组合可以发现我们最多可以生成25个不同的nft，而这些nft本身是无法进行良好的流动性交易的，我们需要的流动性交易是一种更接近同质化的代币，而这种代币实际上就在这10个trait中

3.3 做法：我们可以将10个trait，每一个trait拆分为多个碎片，实际上，每一个trait图片、属性完全相同，可以等同看作一个ERC20代币，基于这样产生后的NFT碎片，非同质化属性大大降低
例：我们可以将trait A拆分为A1…A10，这里名字不同仅作区分，实际上碎片是完全相同的，基于这样子的碎片拆分后，我们可以将其与ERC20代币进行组合

4、通过这样的方式，这10个trait可以按照需求拆分为x*10个碎片（x为每一个trait的碎片数量），基于这样的设计方式，我们可以将这些同质化后的trait碎片组成流动性池，并以reserve（erc20）或其它方式组成相应的market maker算法，完成nft流动性的创建

**Comment by luckyyang** [comment]
有意思，这样的话，某个 AMM pool 就可以是 100 个child A trait + 100 USDC 这样的组合？

**Reply by BradMoonUESTC** [reply]
是的 还在设计中 会参考1155和3664

**Reply by luckyyang** [reply]
期待更多的进展

**Comment by luckyyang** [comment]
看到这个 mirror 上的文章 [关于NFT流动性的研究](https://mirror.xyz/0x0F9C6A9d8f9729aB07b5b2B870Ce8122E76708b0/6CGlYphepRspxrFoTGcESaDvH7rVrz4pvGzONaHy4V8)

**Reply by BradMoonUESTC** [reply]
嗯呢，我看看，最近忙科研的事情没时间推进T_T

**Reply by BradMoonUESTC** [reply]
链接有点问题，在mirror找到了，我更新一下: [关于NFT流动性的研究](https://mirror.xyz/0x0F9C6A9d8f9729aB07b5b2B870Ce8122E76708b0/6CGlYphepRspxrFoTGcESaDvH7rVrz4pvGzONaHy4V8)

**Reply by luckyyang** [reply]
粘贴错了，我也更新了，感谢
